### PR TITLE
GetBalance for non-nullified commitments.

### PR DESCRIPTION
**Pull request by kKahina**
getBalance return the value of the non-nullified commitments.

### CODE REVIEW

File: src/traverse/Scope.ts
Comment (SwatiEY) [inline src/traverse/Scope.ts:1]: we are not using the isPublic key. if it is not used, can you please remove it

File: src/codeGenerators/orchestration/files/toOrchestration.ts
Comment (SwatiEY) [inline src/codeGenerators/orchestration/files/toOrchestration.ts:1]: please remove the unnecessary blank lines

File: src/boilerplate/orchestration/javascript/nodes/boilerplate-generator.ts
Comment (SwatiEY) [inline src/boilerplate/orchestration/javascript/nodes/boilerplate-generator.ts:1]: Remove the unnecessary blank lines

File: src/boilerplate/common/web3.mjs
Comment (SwatiEY) [inline src/boilerplate/common/web3.mjs:1]: Please remove the blank lines

File: src/boilerplate/common/commitment-storage.mjs
Comment (SwatiEY) [inline src/boilerplate/common/commitment-storage.mjs:1]: Please remove the blank lines

**Review by SwatiEY — CHANGES_REQUESTED** [review]

**Review by kKahina — COMMENTED** [review]
I removed the blank lines

File: src/boilerplate/common/commitment-storage.mjs
Comment (lydiagarms) [inline src/boilerplate/common/commitment-storage.mjs:1]: Could you translate the comments to English?

**Review by lydiagarms — APPROVED** [review]

**Review by SwatiEY — APPROVED** [review]

**Pull request merged**